packages/prop-house-webapp/src/components/ProposalModal/index.tsx

```diff
@@ -239,8 +234,8 @@ const ProposalModal = () => {
       {showDeletePropModal && activeProposal && (
         <DeleteProposalModal
           id={activeProposal.id}
-          showModal={showDeletePropModal}
-          setShowModal={setShowDeletePropModal}
+          showDeletePropModal={showDeletePropModal}
+          setShowDeletePropModal={setShowDeletePropModal}
           setEditProposalMode={setEditProposalMode}
```

Comment:
Not sure why this is a prop. Presumably, you never see the `DeletePropModal` when in editing mode? Also, not sure why you would manage whether or not you are in editing while seeing the `DeletePropModal`

Reply:
you're right, must've been old code. removed: https://github.com/Prop-House/prop-house-monorepo/pull/386/commits/e09ec0c6cf1d0b80f41ff2b458e77a5884f41ec6